<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StaticObjectEditor</class>
 <widget class="QWidget" name="StaticObjectEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Static Object Editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="8" column="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Offset</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Size</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="arrValName"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2" colspan="2">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="8" column="3">
      <widget class="QSpinBox" name="arrOffset">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="maximum">
        <number>2147483647</number>
       </property>
      </widget>
     </item>
     <item row="9" column="3">
      <spacer name="verticalSpacer_8">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>261</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="3">
      <widget class="QSpinBox" name="arrValue">
       <property name="minimum">
        <number>-2147483647</number>
       </property>
       <property name="maximum">
        <number>2147483647</number>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QComboBox" name="arrType">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>uint8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>uint16</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>uint32</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>int8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>int16</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>int32</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>bool</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>pointer</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>vector2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>text</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>animator</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>hitbox</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>unknown</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>Value</string>
       </property>
      </widget>
     </item>
     <item row="7" column="3">
      <widget class="QSpinBox" name="arrSize">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="maximum">
        <number>2147483647</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Object Name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" rowspan="9" colspan="2">
      <widget class="QTreeView" name="arrTree">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="objName"/>
     </item>
     <item row="0" column="2" colspan="2">
      <widget class="QPushButton" name="expCHdr">
       <property name="text">
        <string>Export Static Object to C Header</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionAboutRetroED">
   <property name="text">
    <string>About RetroED</string>
   </property>
  </action>
  <action name="actionChangeExecutable">
   <property name="text">
    <string>Change executable</string>
   </property>
  </action>
  <action name="actionRunExecutable">
   <property name="text">
    <string>Run executable</string>
   </property>
  </action>
  <action name="actionRSDKUnpacker">
   <property name="text">
    <string>RSDK Unpacker</string>
   </property>
  </action>
  <action name="actionSceneEditor">
   <property name="text">
    <string>Scene Editor</string>
   </property>
  </action>
  <action name="actionScriptEditor">
   <property name="text">
    <string>Script Editor</string>
   </property>
  </action>
  <action name="actionGameconfigEditor">
   <property name="text">
    <string>Gameconfig Editor</string>
   </property>
  </action>
  <action name="actionGFXTool">
   <property name="text">
    <string>GFX Tool</string>
   </property>
  </action>
  <action name="actionRSVExtractor">
   <property name="text">
    <string>RSV Extractor</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen_2">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave_2">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
